modules/lss-creditserweiterung/creditserweiterung.user.js

```diff
@@ -0,0 +1,220 @@
+(($, win, I18n) => {
+    I18n.translations.de.lssm.creditserweiterung = {
+        setCookieRegex1: /<\/?[^>]+>/gi,
+        setCookieRegex2: /(.|\n)*Dienstgrad: /g,
+        setCookieRegex3: / verdiente(.|\n)*/g,
+        setCookieRegex4: /(\d|.)*\n../g,
+        setCookieRegex5: /\./g,
+        setCookieRegex6: /\n.*/g,
+        earnedCreditsCookie: 'gesamtcredits',
+        rankCookie: 'dienstgrad',
+        creditsNeeded: [
```

Comment:
Wenn ich es richtig sehe, dann ist das nicht sprachabhängig und könnte in einen globalen Scope.

Reply:
nein, die unterschiedlichen Spiele haben unterschiedliche Ränge, von den Creditszahlen, die benötigt werden her, auch in der Anzahl der Ränge unterscheiden sie sich

Reply:
Ja, schon klar. Ich meinte auch **eine** pro Sprache. Momentan machst du das mit 6 Teil-Regexen. Mit einer wäre das sehr viel übersichtlicher.

Reply:
Z.B. so

var results = $('.page-header').text().match(/(.*)([0-9]\.)? verdiente Credits.*/i);
var credits = results[1];

Reply:
vielen dank für diese Hilfestellung ;)

Reply:
![image](https://user-images.githubusercontent.com/25933231/34325466-83d73d58-e892-11e7-88b3-ae3d595ce5d8.png)

Reply:
Ist der Unterschied klar? Lieber das extrahieren was du brauchst, statt abzuschnibbeln was du nicht brauchst.

Reply:
ja, hört sich sowieso sinnvoller an, als meine Lösung :D